frontend/public/components/storage-class-form.tsx

```diff
@@ -648,40 +573,60 @@ export class StorageClassForm_ extends React.Component<
       : this.setState({ newStorageClass: newParams });
   };
 
+  addDefaultParams = () => {
+    const defaultParams = this.state.newStorageClass.type
+      ? this.storageTypes[this.state.newStorageClass.type].parameters
+      : {};
+    const hiddenParmas = {};
+    _.each(defaultParams, (values, param) => {
```

Comment:
```suggestion
    Object.entries(defaultParams).forEach((item) => {
    // item[0] is the key , item[1] is the value
```

Reply:
@bipuladh, the code will look little messy if we do this
```
Object.entries(defaultParams).forEach((item) => {
      if (item[1]['visible'] && !item[1]['visible']() && item[1]['value']) {
        hiddenParmas[item[0]] = item[1];
      }
    });
```
Shall we keep it same, wdyt?

Reply:
yup let's resolve this.